### PR TITLE
GA events for demo click added

### PR DESCRIPTION
**Pull request by vivek-devtron**
GA events added for demo click on header and main page

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -134,7 +134,7 @@
                 <a href="team.html" aria-current="page" class="nav-link white w-nav-link">Team</a>
                 <a href="careers.html" class="nav-link white w-nav-link">Careers</a>
                 <a href="https://devtron.ai/blog/" class="nav-link white w-nav-link" target="_blank">Blog</a>
-                <a href=" https://preview.devtron.ai/dashboard/" class="demo-button tab-view" target="_blank">Demo <img src="images/arrow-right.svg"></a>
+                <a href=" https://preview.devtron.ai/dashboard/" class="demo-button tab-view" target="_blank" onclick="demoHeader('index')">Demo <img src="images/arrow-right.svg"></a>
```
Comment (sohel-rp) [inline public/index.html:137]: same here

File: public/index.html [hunk 2]
```diff
@@ -144,7 +144,7 @@
                   </div>
                 </a>
               </nav>
-              <a href=" https://preview.devtron.ai/dashboard/" class="demo-button margin-left" target="_blank">Demo <img src="images/arrow-right.svg"></a>
+              <a href=" https://preview.devtron.ai/dashboard/" class="demo-button margin-left" target="_blank" onclick="demoHeader('index')">Demo <img src="images/arrow-right.svg"></a>
```
[the remark this answers is not in the document]
Comment (vivek-devtron) [reply public/index.html]: Done

File: public/index.html (hunk 1 above)
Comment (vivek-devtron) [reply public/index.html]: Done

**Review by sohel-rp — APPROVED** [review]
Looks good 👍

**Pull request merged**